CLI slides

decklin opened the pull request:
Did a presentation to cover what's in the command line page of the upstream docs, plus a few more commands, and concepts and history. Probably still too much of an info-dump. Moved the old command line page to a sort of "extra credit" section since it had some unrelated stuff, so we can take whatever needs to be cut for time and move it there. Feedback appreciated! Anyone else is of course welcome to make major edits or run with presenting it.

decklin committed:
Add CLI presentation slides (a bit long, but it's a starting point)

decklin committed:
Remove orphaned command_prompt page

decklin committed:
Link to CLI presentation, move old page to extra credit

decklin committed:
Update deckrb and dependencies (required for .notes)

mxie commented on views/workshop/command_line.deck.markdown:
```diff
@@ -0,0 +1,362 @@
+# Introduction to the command line
+
+Decklin Foster  
+Railsbridge Boston  
```
`RailsBridge` ;)

decklin replied:
heh, of course I miss it in the title slide :)

mxie commented on views/workshop/command_line.deck.markdown:
```diff
@@ -0,0 +1,362 @@
+* **On Macs:** In Finder, start Applications > Utilities > Terminal  
+  or find the Terminal application through Spotlight (click the magnifying glass in the top right of the screen and start typing 'Terminal')"
+* **On Windows:** open up Git Shell from your desktop or All Programs menu.
+* **On Linux:** press Ctrl + Alt + T or Find Terminal under the Accessories category of your applications menu.
```
Perhaps add a line for those using a Chromebook:

```
**On Chromebook:** From the [boxes page](https://www.nitrous.io/app#/boxes), click the "IDE" button. The command line window should appear at the bottom of the screen under the "Console" tab.
```

It looks like this, if you're curious:
![nitrous io 2014-09-02 14-26-26 2014-09-02 14-26-58](https://cloud.githubusercontent.com/assets/602470/4123488/b9f0d138-32ce-11e4-86e2-dcdb23b683ea.jpg)

mxie replied:
I'll update this on the other repo as well.

decklin replied:
Cool, thanks

decklin committed:
Fix several occurences of RailsBridge where the B was not capitalized

decklin committed:
Add a line for finding the terminal equivalent for Chromebook (nitrous.io)

Suggested by Melissa. Can this terminal actually do all the things that
we do in the presentation? I'm not sure.

decklin commented:
updated based on comments.

mxie commented:
Everything looks great!

To address your concern about the terminal functionality, it's actually just a Linux box that has all the Rails software installed for the user, so I think it'll work fine.

Feel free to rebase and merge when you're ready.

mxie commented on views/workshop/command_line.deck.markdown:
```diff
@@ -0,0 +1,363 @@
+
+[http://docs.railsbridge.org/ruby/command_line](http://docs.railsbridge.org/ruby/command_line)
+
+.notes TODO: This is is a link to railsbridge.org! We should probably merge and deploy the updated docs to railsbridgeboston.org, then change the link.
```
FYI, there's a discussion "going on" about separating the command line material as its own curriculum: https://github.com/railsbridge/docs/issues/335

mxie replied:
Also, have you considered submitting these slides over in https://github.com/railsbridge/docs/tree/master/sites/en/workshop?

decklin commented:
I'll take a look! Merging this now.
